src/components/app/details/triggerView/TriggerView.utils.ts

```diff
@@ -84,7 +124,10 @@ export const processResolvedPromise = (resp: { status: string; value?: any; reas
             wfrId: resp.value?.result?.wfrId,
         }
     }
-
+    //This will handle a corner case, no previous deployment history is present i.e for first time deployment.
+    if (isRecentDeployConfig) {
```

Comment:
can we do if (!isRecentDeployConfig) { showError(resp.reason)}

Reply:
done